### PR TITLE
python3.pkgs.mediapipe: init at 0.9.2.1

### LINKED ISSUE #292509: Package request: python.mediapipe
**Project description**
MediaPipe is the simplest way for researchers and developers to build world-class ML solutions and applications for mobile, edge, cloud and the web.

**Metadata**

* homepage URL: https://pypi.org/project/mediapipe/
* source URL: https://github.com/google/mediapipe
* license: Apache 2.0
* platforms: unix, linux, darwin

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

### PR DESCRIPTION
**Pull request by DerDennisOP**
###### Description of changes
cp from NixOS/nixpkgs/pull/196708
changes: 0.8.1 -> 0.9.2.1 && removed binary-hashes.nix

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.05 Release Notes (or backporting 22.11 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2305-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

fixes https://github.com/NixOS/nixpkgs/issues/292509

### CODE REVIEW

**Review by fabaff — COMMENTED** [review]
The empty file https://github.com/NixOS/nixpkgs/pull/225901/files should be removed.

File: pkgs/development/python-modules/mediapipe/default.nix [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  version = "0.9.2.1";
+  disabled = pythonOlder "3.8" || pythonAtLeast "3.11";
+  format = "wheel";
+
+  poseHeavy = fetchurl {
+    url = "https://storage.googleapis.com/mediapipe-assets/pose_landmark_heavy.tflite";
+    sha256 = "WeQtcbzUTL26vEGfD/dmhllf0mVBlWa9QAnvcD6o4f4=";
+  };
+
+  src =
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/mediapipe/default.nix:31]: Please take a look at https://github.com/NixOS/nixpkgs/blob/c58e6fbf258df1572b535ac1868ec42faf7675dd/pkgs/development/python-modules/home-assistant-chip-core/default.nix#L32-L51 to make this work with multiple python versions and system

Comment (DerDennisOP) [reply pkgs/development/python-modules/mediapipe/default.nix:31]: I'm not quite sure how

**Comment by AidanJ** [comment]
Is the merge request still pending review?

File: pkgs/development/python-modules/mediapipe/default.nix [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+}:
+
+buildPythonPackage rec {
+  pname = "mediapipe";
+  version = "0.9.2.1";
+  disabled = pythonOlder "3.8" || pythonAtLeast "3.11";
+  format = "wheel";
+
+  poseHeavy = fetchurl {
+    url = "https://storage.googleapis.com/mediapipe-assets/pose_landmark_heavy.tflite";
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/mediapipe/default.nix:27]: We should probably upload this to web.archive.org just to be safe

**Pull request closed without merging**